app/templates/service_update_audits.html

```diff
@@ -139,26 +139,19 @@ <h3 class="validation-masthead-heading" id="validation-masthead-heading">
               {% endcall %}
             {% endcall %}
 
-            <!--
-            <nav role="navigation">
-                <ul class="previous-next-navigation">
-
-                    <li class="previous">
-                        <a href="">
-                            Previous <span class="visuallyhidden">page</span>
-                            <span class="page-numbers">1 of 3</span>
-                        </a>
-                    </li>
-
-                    <li class="next">
-                        <a href="">
-                            Next <span class="visuallyhidden">page</span>
-                            <span class="page-numbers">2 of 3</span>
```

Comment:
I don't think we have any way of knowing how many pages there are in total. But we can definitely tell the user what the next/previous page is:

514e3bf

![image](https://cloud.githubusercontent.com/assets/355079/9719477/82925316-557c-11e5-8a35-589604c4c4b2.png)